.github/.gitignore

```diff
@@ -0,0 +1 @@
+*.html
```

Comment:
why do we need to ignore html? Is this to ignore doc build outputs? If so, can we limit this to some known directory? I can imagine wanting to add some other html files later on.